### ISSUE TITLE
Please show how to use it as soon as possible

### ISSUE THREAD

**Issue by UCAS-Vincent** [xhu248/semi_cotrast_seg#1]
Please show how to use it as soon as possible

**Comment by UCAS-Vincent** [comment]
Please show how to use it as soon as possible

**Comment by xhu248** [comment]
please see the latest repository, thanks!

**Comment by JohnMiao1** [comment]
It seems that the preprocessing.py code has something wrong in Line 42 "image = image[:, :, 0].transpose((0, 2, 1))". The dimension is not matched.

**Comment by zwjulian** [comment]
Did you already find a solution JohnMiao1? I have the same issue.

**Comment by JohnMiao1** [comment]
Nope. I changed the code according to my understanding, but got a bad result.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xhu248/semi_cotrast_seg"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月23日(星期三) 凌晨2:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [xhu248/semi_cotrast_seg] Please show how to use it as soon as possible (#1)





 
Did you already find a solution JohnMiao1? I have the same issue.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by wangyihub** [comment]
I have a question tha:i am not sure that the percent of labeled data to use in ablation experiments in table 1 is pionted in  step 2 (pixel contrast in supervised learing)  ,or step 3(fine tune).
